cypress/utils/dashboards/remove_workspace.js

```diff
@@ -7,7 +7,7 @@ export function removeSampleDataAndWorkspace(url, workspaceName) {
   describe('removing workspace/sampledata', () => {
     it('remove workspace', () => {
```

Comment:
not specific to this pr but how come this a utils file if it's a test ?

Reply:
I believe because this can be re-used across a number of tests. For example, I write a new test and I want to remove a workspace as a part of this test and get solid logs of which parts failed or succeed as a part of that process.